models/user.go

```diff
@@ -458,10 +458,11 @@ func (u *User) ContainsTeamId(id int64) (bool, int) {
 	return false, -1
 }
 
-// Update an array of users.
+// UpdateUsers updates an array of users.
+//
 func UpdateUsers(c appengine.Context, users []*User) error {
 	keys := make([]*datastore.Key, len(users))
-	for i, _ := range keys {
+	for i := range keys {
```

Comment:
After looking at the Go doc, my fix is correct:

![image](https://cloud.githubusercontent.com/assets/1008617/12778083/09bdd8d4-ca61-11e5-8d60-d7cc671b592f.png)